RNASeqTools_TSS/rnaseqtools_tss.jl

```diff
@@ -0,0 +1,105 @@
+# # RNASeq data analysis using RNASeqTools
+#
+# Requirements:
+#
+# * Julia 1.8.5
+# * RNASeqTools 0.3.4
+# * sra-tools 3.0.3
+# * samtools 1.6
+# * bwa-mem2 2.2.1
```

Comment:
If they're not already, we should get these set up on BinaryBuilder

Reply:
If these are external dependencies not managed by julia's package manager, it could be complicated to include.

One possibility is to use `CondaPkg`, but getting the jll's would be better, I think

Reply:
Having all the binary dependencies packaged would be great! I checked and could find samtools and BWA (but not BWA-MEM2, which is faster) artifacts, but none for sra-tools. I also had a quick look into BioServices.jl and could see a way to search for sra accession numbers, but not to download the actual data.

I will read a bit into BinaryBuilder to see what it takes to add something. I can imagine sra-tools to be a nightmare to put into a package since it has so many individual tools. Maybe prefetch and fastq-dump could be in their own artifacts...